src/expr.rs

```diff
@@ -781,10 +797,26 @@ fn rewrite_if_else(context: &RewriteContext,
                                else_block.span.lo);
         let after_else_comment = extract_comment(after_else, &context, offset, width);
 
+        let between_sep = match context.config.else_if_brace_style {
+            ElseIfBraceStyle::AlwaysNextLine |
+            ElseIfBraceStyle::ClosingNextLine => alt_block_sep,
+            _ => " ",
```

Comment:
Nittiest of nits: there's only one case left here, right? I think it's more idiomatic to provide that instead of `_`.